openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -371,14 +371,19 @@ def process_batch(self, media_batch):
 
         return record_count
 
-    @abstractmethod
     def get_media_type(self, record: dict) -> str:
         """
         For a given record, return the media type it represents (eg "image", "audio",
-        etc.) If a provider only supports a single media type, this may be hard-coded
-        to return that type.
+        etc.) If a provider only supports a single media type, this method defaults
+        to returning the only media type defined in the ``providers`` attribute.
         """
-        pass
+        if len(self.providers) == 1:
+            return list(self.providers.keys())[0]
+
+        raise NotImplementedError(
+            "Provider scripts that support multiple media types "
+            "must provide an override for ``get_media_type``."
+        )
```

Comment:
I _love_ this! 😄 Can you update the new `ProviderDataIngester` docs and template?

Out of scope, but should we make an issue to clean up our other provider scripts (and their tests)?

Reply:
> Out of scope, but should we make an issue to clean up our other provider scripts (and their tests)?

Yes, in a separate issue, for sure! I can do that.

Reply:
Done: https://github.com/WordPress/openverse-catalog/issues/822